src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMoveBookingIntTest.kt

```diff
@@ -250,14 +253,72 @@ class PrisonPersonMoveBookingIntTest : SqsIntegrationTestBase() {
 
         // the from offender is sync'd from NOMIS to DPS
         physicalAttributesDpsApi.verify(
-          putRequestedFor(urlPathEqualTo("/sync/prisoners/$fromOffenderNo/profile-details-physical-attributes")),
+          putRequestedFor(urlPathEqualTo("/sync/prisoners/$fromOffenderNo/profile-details-physical-attributes"))
+            .withRequestBodyJsonPath("build.value", "MEDIUM"),
         )
 
         // the to offender IS sync'd from NOMIS to DPS
         physicalAttributesDpsApi.verify(
           count = 1,
+          putRequestedFor(urlPathEqualTo("/sync/prisoners/$toOffenderNo/profile-details-physical-attributes"))
+            .withRequestBodyJsonPath("build.value", "LARGE"),
+        )
+
+        // the to offender is also sync'd back from DPS to NOMIS
+        nomisSyncApi.verify(
+          putRequestedFor(urlPathEqualTo("/prisonperson/$toOffenderNo/physical-attributes")),
+        )
+
+        // telemetry
+        verify(telemetryClient).trackEvent(
+          eq("prisonperson-booking-moved"),
+          check {
+            assertThat(it).containsExactlyInAnyOrderEntriesOf(
+              mapOf(
+                "bookingId" to movedBookingId.toString(),
+                "toOffenderNo" to toOffenderNo,
+                "fromOffenderNo" to fromOffenderNo,
+                "syncFromOffenderDps_BUILD" to "true",
+                "syncToOffenderDps_BUILD" to "true",
+                "syncToOffenderNomis" to "true",
+              ),
+            )
+          },
+          isNull(),
+        )
+      }
+
+      @Test
+      fun `mix of offender profile details re-entered and not re-entered`() {
```

Comment:
Added another profile details test where there is a mixture of details updated / not updated